docs/content/en/docs/install/install.md

```diff
@@ -60,14 +63,28 @@ Some helpful hints:
 * Use the `--version <version>` flag on the
   `helm upgrade --install` command to specify a different Keptn Helm chart version.
 
-* Use the following command sequence to see a list of available versions:
+  To get the appropriate chart version for the Keptn version you want,
+  use the following command sequence:
 
   ```shell
+  helm repo add keptn https://charts.lifecycle.keptn.sh
```

Comment:
this line is not needed since it was in the initial install instructions

Reply:
And then we are relying on the user to think things through a bit.  This was also @bacherfl 's suggestion -- if you two agree, I'm good.

Reply:
If a user reruns this line, nothing bad will happen so I would be fine with having it duplicated

Reply:
But if I want to install an older version (let's fast-forward to when v0.10.0 is the latest and I want to install v0.9.2), I actually need to get the chart version before I do my first `helm update...` command sequence.   I guess I could put prose before this, saying that they need to do the two `helm repo` commands show for installation before looking up he chart version but that seems more onerous than just putting those lines in the example.  But if you disagree, I'll remove them.

Reply:
I removed the lines